src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -872,8 +872,16 @@ namespace NServiceBus
     }
     public class static ScheduleBusExtensions
     {
-        public static void ScheduleEvery(this NServiceBus.IBusContext bus, System.TimeSpan timeSpan, System.Action task) { }
-        public static void ScheduleEvery(this NServiceBus.IBusContext bus, System.TimeSpan timeSpan, string name, System.Action task) { }
+        [System.ObsoleteAttribute("Please use `ScheduleEvery(this IBusContext context, TimeSpan timeSpan, Func<Task>" +
+            " task)` instead. Will be treated as an error from version 7.0.0. Will be removed" +
+            " in version 8.0.0.", false)]
+        public static void ScheduleEvery(this NServiceBus.IBusContext context, System.TimeSpan timeSpan, System.Action task) { }
+        [System.ObsoleteAttribute("Please use `ScheduleEvery(this IBusContext context, TimeSpan timeSpan, string nam" +
```

Comment:
`ScheduleEvery(this NServiceBus.IBusContext context, System.TimeSpan timeSpan, string name, System.Func<NServiceBus.IBusContext, System.Threading.Tasks.Task> task)`